packages/evm/src/evm.ts

```diff
@@ -137,9 +137,15 @@ export class EVM implements EVMInterface {
 
   protected readonly _emit: (topic: string, data: any) => Promise<void>
 
-  constructor(opts: EVMOpts = {}) {
-    this.events = new AsyncEventEmitter()
+  private bn128: {
```

Comment:
Does it make sense to create a `EVM.create` method similar to `VM.create`?

Reply:
That thought definitely occurred to me while I was doing this. Let me take a look at our current usage of the EVM constructor and see if it makes sense. It would definitely simplify the API